terraform/push2codecommit.tf

```diff
@@ -0,0 +1,23 @@
+#data "aws_caller_identity" "current" {}
+resource "null_resource" "push_petcliniccode" {
+  provisioner "local-exec" {
+    command = <<EOT
+	cd ../petclinic/
+	git config --global --unset credential.helper
+	git config --system --unset credential.helper
+	git config --global user.name "Imtranur Rahman"
+	git config --global user.email imtranur@amazon.com
```

Comment:
Do we want to include personal information here? It's committing to a CodeCommit repo, so it doesn't really matter what we put here

Reply:
good catch let me make it a variable